x/accesscontrol/keeper/keeper.go

```diff
@@ -317,15 +330,17 @@ func (k Keeper) ResetWasmDependencyMapping(
 	contractAddress sdk.AccAddress,
 	reason string,
 ) error {
-	dependencyMapping, err := k.GetWasmDependencyMapping(ctx, contractAddress, "", []byte{}, false, make(ContractReferenceLookupMap))
+	dependencyMapping, err := k.GetRawWasmDependencyMapping(ctx, contractAddress)
 	if err != nil {
 		return err
 	}
 	store := ctx.KVStore(k.storeKey)
 	// keep `Enabled` true so that it won't cause all WASM resources to be synchronous
-	dependencyMapping.AccessOps = types.SynchronousAccessOpsWithSelector()
+	dependencyMapping.BaseAccessOps = types.SynchronousWasmAccessOps()
```

Comment:
maybe we simply delete the key instead such that it returns synchronous by default as the behavior in GetWasmDependencyMapping to return sync access ops if no registered deps exist?

Reply:
I guess it could make sense to keep this to persist the reset reason, but couldn't we expose the reset reason via error logging instead, or is that not good enough since the logs would be volatile instead of persistent?

Reply:
I was thinking about letting the contract owner know why their contract gets reset. If the documentation makes that clear enough though I think deleting the key could work (and better)

Reply:
yeah, this sounds fine, we can keep it in that case